testsuite/integration-arquillian/tests/other/console/pom.xml

```diff
@@ -32,7 +32,8 @@
     <name>Admin Console UI Tests</name>
 
     <properties>
-        <keycloak.theme.dir>${auth.server.home}/themes</keycloak.theme.dir>
+        <keycloak.theme.dir>${auth.server.home}/themes-primary</keycloak.theme.dir>
+        <developer.theme.dir>${project.basedir}/../../../../../themes/src/main/resources/theme</developer.theme.dir>
```

Comment:
Is this not changing the server that is tested? There seems to be a chance here that resources not copied by mistake when building the server would be loaded directly from themes/src and the bug wouldn't be caught during testing.